packages/Graph/Schemas/Microsoft.Graph.User.GetDirectReports.schema

```diff
@@ -36,6 +36,14 @@
         "dialog.worksWithMe.maxResults"
       ]
     },
+    "fieldsToSelect": {
+      "$ref": "schema:#/definitions/stringExpression",
+      "title": "Fields To Select",
+      "description": "The fields of from the graph API to select. By default, if not provided only the ID will be returned for the object.",
+      "examples": [
+        "dialog.fieldsToSelect"
```

Comment:
Nit: might be nice to include examples here and also a link to where the field names can be found for people trying to use this for other use cases. Not critical though

Reply:
That's good idea! I've updated the description and examples to reflect that.

![image](https://user-images.githubusercontent.com/9922190/113766960-ebe6c480-96d2-11eb-9931-d95c30439f03.png)